testing/internal/e2e/vault/vault.go

```diff
@@ -3,17 +3,20 @@ package vault
 
 import (
 	"context"
+	"embed"
 	"fmt"
 	"os"
-	"path"
-	"runtime"
 	"testing"
 
 	"github.com/hashicorp/boundary/testing/internal/e2e"
+	"github.com/hashicorp/go-secure-stdlib/base62"
 	"github.com/kelseyhightower/envconfig"
 	"github.com/stretchr/testify/require"
 )
 
+//go:embed boundary-controller-policy.hcl
```

Comment:
@johanbrandhorst I tried using the embed here. It wasn't working with a `testdata` path, presumably because this particular package isn't a test package?

Reply:
If it's in testdata, you may need to reference testdata explicitly
```suggestion
//go:embed testdata/boundary-controller-policy.hcl
```

This would also affect the path when calling `ReadFile`

Reply:
Yup. I was missing the `ReadFile` part when I was trying it earlier.